src/Modal/Modal.component.tsx

```diff
@@ -0,0 +1,271 @@
+// REACT
+import * as React from 'react';
+
+// ANCHOR
+import { Typography } from '..';
+import { Close } from '../Icon';
+import { IconSVGProps } from '../Icon/utils';
+import { sizes } from '../theme/sizes.theme';
+import { colors } from '../theme/colors.theme';
+
+// VENDOR
+import * as StyledReactModal from 'styled-react-modal';
+import styled, { css } from 'styled-components';
+import classnames from 'classnames';
+import { opacify } from 'polished';
+
+export type ModalSize = 'lg' | 'sm';
+
+const Sizes = {
```

Comment:
I noticed that wherever these int's were being used, ultimately they were being interpolated  to rem's. Is there a reason not to specify the unit type here?

Reply:
No real reason in this PR. In the button PR I ended up switching from strings to ints a few times so that I could more easily do math on the values, so this time I just started with ints.